src/internal/middleware/auth/interceptor.go

```diff
@@ -224,6 +224,12 @@ var authHandlers = map[string]authHandler{
 
 	// TODO: Only the pachd sidecar instances should be able to use this endpoint.
 	"/proxy.API/Listen": unauthenticated,
+
+	//
+	// Task API
+	//
+
+	"/taskapi.API/ListTask": authDisabledOr(authenticated),
```

Comment:
unfortunately the test I wrote didn't catch that I missed adding the interceptor